services/CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
+# 0.3 (2022-04)
+
+- [#732](https://github.com/Azure/azure-sdk-for-rust/pull/732) generated from azure-rest-api-specs [commit from 2022-04-27](https://github.com/Azure/azure-rest-api-specs/commit/46ffdc4fe0f9a413ba29ed859b5ff4174ce1c7ec)
+- [#718](https://github.com/Azure/azure-sdk-for-rust/pull/718) include `path` and `x_ms_path` endpoints
+- [#706](https://github.com/Azure/azure-sdk-for-rust/pull/706) update `azure_core` to 0.2
```

Comment:
This actually happened in the 0.2 release, but not a big deal. 
https://crates.io/crates/azure_mgmt_resources/0.2.0/dependencies

Reply:
The 0.2 release was actually early 2022-04.